docs/facts/schema.md

```diff
@@ -55,10 +59,12 @@ attribute((field,unique),f2,111).
 attribute((field,min),f2,0).
 attribute((field,max),f2,55).
 attribute((field,std),f2,6).
+attribute((field,entropy),f1,2.4218).
```

Comment:
Hmm, this doesn't parse for me. 

<img width="744" alt="Screen Shot 2022-05-06 at 7 15 58 PM" src="https://user-images.githubusercontent.com/589034/167226974-378c991a-47a9-4250-a732-e34fdb28d045.png">

Reply:
Yeah, I realized that when I wrote soft constraints with entropy. I wonder how I can fix this issue? Should I change the entropy to int instead?

Reply:
I think you can use the string format

Reply:
String format won't let us do comparisons. I think we need to use int.